csharp/ql/lib/semmle/code/csharp/ExprOrStmtParent.qll

```diff
@@ -140,7 +140,9 @@ private module Cached {
   }
 
   private Expr getAChildExpr(ExprOrStmtParent parent) {
-    result = parent.getAChildExpr() or
+    result = parent.getAChildExpr() and
+    not result = parent.(DeclarationWithGetSetAccessors).getExpressionBody()
```

Comment:
Why is expression body filtered out?

Reply:
Because, as far as enclosing callable is concerned, the parent should be the getter `get_P` and not the property `P` in
```csharp
public int P => foo;
```